stable/odoo/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: odoo
-version: 3.2.2
+version: 3.2.3
```

```suggestion
version: 4.0.0
```

As there is a big change in the dependency I think we should also update the major version in the parent chart.